packages/flow-lineage/src/components/f-lineage/create/create-node-elements.ts

```diff
@@ -120,12 +120,14 @@ export default function createNodeElements(
 				/**
 				 * checking level max Y
 				 */
-				const maxYWhenScrollBar = nodeElement.y + nodeSize.height + maxChildrenHeight;
-				if (nodeElement.hasScrollbaleChildren && levelPointer.y > maxYWhenScrollBar) {
-					levelPointer.maxY = maxYWhenScrollBar;
-					nodeElement.childrenYMax = maxYWhenScrollBar;
-				} else if (levelPointer.y > (levelPointer.maxY ?? 0)) {
-					levelPointer.maxY = levelPointer.y;
+				if (!nodeElement.fHideChildren) {
+					const maxYWhenScrollBar = nodeElement.y + nodeSize.height + maxChildrenHeight;
+					if (nodeElement.hasScrollbaleChildren && levelPointer.y > maxYWhenScrollBar) {
```

Comment:
`hasScrollableChildren`

Reply:
hasScrollableChildren is true if maxChildHeight is greater than the actual children's height and the second part is checking if the current y coordinate of a particular level is greater than the total height parent-child (This offset is calculated or used when children are collapsed and opened)